web/src/pages/index.module.css

```diff
@@ -105,3 +105,7 @@
   max-height: 200px;
   max-width: 500px;
 }
+
+[data-theme="dark"] .svgWrapper svg {
+  filter: invert(100%) hue-rotate(180deg);
+}
```

Comment:
We have already a definition in our global CSS file. Should we perhaps just reuse that class definition? Or is there a benefit of doing that here separately?

Reply:
(Your fixups work like a charm otherwise, by the way. I've enabled auto-merge to to signal we're done minus this thread.)

Reply:
Okay, I have reused the `markdown-inline-svg` class.